sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -6,10 +6,27 @@
 
 ### Breaking Changes
 
+- Merged `MultiCategoryClassifyResult` and `SingleCategoryClassifyResult` into `LabelClassifyResult`.
+- Removed Extractive Text Summarization and related models. To access this beta feature, install the 5.2.0-beta.3 version of the client library.
+- Removed Extractive Text Summarization, Multi Label Classification, and Single Label Classification from legacy client library.
+- Removed `HealthcareTaskParameters.FhirVersion` and `AnalyzeHealthcareEntitiesResult.FhirBundle`. To access this beta feature, install the 5.2.0-beta.3 version of the client library.
+- Renamed `LabelClassifyResult.Classifications` to `LabelClassifyResult.ClassificationCategories`.
```

Comment:
We kept this as `classifications` in Python, but we prefer shorter names

Reply:
Java had `ClassifiedCategories` but after an internal discussion, changed to `ClassificationCategories`, so this is in line with Java. Apparently, we like long names.